homeassistant/core.py

```diff
@@ -318,6 +319,14 @@ def async_stop(self) -> None:
         self.state = CoreState.not_running
         self.loop.stop()
 
+    # pylint: disable=no-self-use
+    def _async_exception_handler(self, loop, context):
+        """Handle all exception inside the core loop."""
```

Comment:
Could we add a debug here that also includes the exception? 

https://github.com/python/asyncio/blob/master/asyncio/base_events.py#L1195